### PR TITLE
Updating project services to add workflows and adjusting project structure

### PR DESCRIPTION
**Pull request by picksitquick**
## Proposed Changes

  - Updating project folder structure to run workflows.
  - Adjusting pom file to add missing dependencies.
  - Testing APIs endpoint.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,134 +1,162 @@
...
-        </dependency>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.7.1</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.gamedoora.backend.projectservices</groupId>
```
Comment (aprajshekhar) [inline pom.xml]: com.gamedoora.backend is the  group id

**Review by aprajshekhar — CHANGES_REQUESTED** [review]
Just one suggestions . Thanks

File: pom.xml (hunk 1 above)
Comment (picksitquick) [reply pom.xml]: Updated, please review. Thanks!

**Review by aprajshekhar — APPROVED** [review]
LGTM

**Pull request merged**